templates/corporate/billing/remote_billing_finalize_login_confirmation.html

```diff
@@ -63,7 +63,7 @@ <h1>Log in to Zulip plan management for<br/>{{ host }}</h1>
                     <label for="remote-billing-confirm-login-tos" class="checkbox">
                         <input id="remote-billing-confirm-login-tos" name="tos_consent" class="required" type="checkbox" value="true" />
                         <span></span>
-                        I agree to the <a href="{{ root_domain_url }}/policies/terms" target="_blank" rel="noopener noreferrer">Terms of Service</a>.
+                        I agree to the <a href="https://zulip.com/policies/terms" target="_blank" rel="noopener noreferrer">Terms of Service</a>.
```

Comment:
I think if someone enables terms of service on their self-hosted system, this change will be wrong.

Under what circumstances is `root_domain_url` not the same as zulip.com in Zulip Cloud? I would expect those to always be identical, such that this change is a noop in Cloud and breaks configuring ToS for a self-hosted system. But I'm probably missing something...

Reply:
oh yeah! I dropped that comment since you are right.